src/Microsoft.FeatureManagement/ConfigurationFeatureDefinitionProvider.cs

```diff
@@ -81,7 +81,7 @@ public async IAsyncEnumerable<FeatureSettings> GetAllFeatureSettingsAsync()
             }
         }
 
-        private FeatureSettings ReadFeatureSettings(string featureName)
+        private FeatureDefinition ReadFeatureSettings(string featureName)
```

Comment:
Do you like to be consistent `ReadFeatureDefinition`?

Reply:
Yes, thank you. I'll update it.